vm/rust/src/lib.rs

```diff
@@ -226,14 +303,15 @@ fn build_block_context(
     chain_id_str: &str,
     block_number: c_ulonglong,
     block_timestamp: c_ulonglong,
-    sequencer_address: StarkFelt,
 ) -> BlockContext {
     BlockContext {
         chain_id: ChainId(chain_id_str.into()),
         block_number: BlockNumber(block_number),
         block_timestamp: BlockTimestamp(block_timestamp),
 
-        sequencer_address: ContractAddress(PatriciaKey::try_from(sequencer_address).unwrap()),
+        // Use default sequencer address
+        // See https://github.com/NethermindEth/juno/pull/901#discussion_r1271858429
+        sequencer_address: ContractAddress::try_from(StarkHash::try_from("0x046a89ae102987331d369645031b49c27738ed096f2789c24449966da4c6de6b").unwrap()).unwrap(),
```

Comment:
This should only be used as a fallback, not the default.
see https://github.com/NethermindEth/juno/pull/896/files#diff-1f724c5e433be46bfd8dc0962d1796f0dce2a5e22253fe82f69ab6a1eb8df278R1168-R1171